l10n_ch_fds_postfinance/wizards/fds_files_import_from_fds_wizard.py

```diff
@@ -113,11 +113,11 @@ def import_button(self):
                 self.process_files(file)
 
             self.state = 'done'
-        except Exception as e:
+        except:
```

Comment:
Are we sure we want to catch all exceptions ? isn't there a subset of exception that are ok during import?